pkg/server/endpoints/node/handler_test.go

```diff
@@ -650,8 +671,18 @@ func getExpectedFetchX509SVID(data *fetchSVIDData) *node.X509SVIDUpdate {
 	caCert, _, _ := util.LoadCAFixture()
 	svidUpdate := &node.X509SVIDUpdate{
 		Svids:               svids,
-		Bundle:              caCert.Raw,
+		DEPRECATEDBundle:    caCert.Raw,
```

Comment:
when we'll remove those deprecated fields? must we add some anotations/comments about a prossible version where we'll remove? or at leas when it was mark as deprecated? so we can track how long we have it?

Reply:
Hmm, i think that is a larger discussion to have about API deprecation paths. For now, is git history sufficient to know when it was deprecated?